### PR TITLE
Feedback component

### PR DESCRIPTION
**Pull request by aziz512**

### CODE REVIEW

**Comment by aziz512** [comment]
capture1

File: src/app/feedback-widget/feedback-widget.component.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<p>
-  feedback-widget works!
-</p>
+<p>Enter your name: <input type="text" #username/></p>
```
Comment (kirjs) [inline src/app/feedback-widget/feedback-widget.component.html]: Please get user name in the input

**Comment by aziz512** [comment]
capture1

File: src/app/feedback-widget/feedback-widget.component.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
-<p>
-  feedback-widget works!
-</p>
+<h2>Give us some feedback!</h2>
+<input type="text" #usernameInput placeholder="Enter your name" required/>  <br>
```
Comment (kirjs) [inline src/app/feedback-widget/feedback-widget.component.html]: don't use br, wrap in divs

File: src/app/feedback-widget/feedback-widget.component.ts [hunk 3]
```diff
@@ -13,8 +13,8 @@ export class FeedbackWidgetComponent implements OnInit {
   ngOnInit() {
   }
 
-  send(comment){
-    this.state.sendFeedback(comment);
+  send(comment, username){
+    this.state.sendFeedback({comment,username});
```
Comment (kirjs) [inline src/app/feedback-widget/feedback-widget.component.ts]: If input is empty, it'll display an error but still send the feedback, or am I missing somethnig?

File: src/app/feedback-widget/feedback-widget.component.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
+*{
```
Comment (kirjs) [inline src/app/feedback-widget/feedback-widget.component.css]: Using * is not the best idea mostly for performance and predictability reasons, just remove this whole section

File: src/app/feedback-widget/feedback-widget.component.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  font-size:20px;
+  font-family: Arial;
+  border-radius:3px;
+}
+textarea{
+  margin:10px 0 10px 0;
```
Comment (kirjs) [inline src/app/feedback-widget/feedback-widget.component.css]: '10px 0' should be enough

File: src/app/feedback-widget/feedback-widget.component.html [hunk 6]
```diff
@@ -1,3 +1,4 @@
...
+<textarea #feedbackInput placeholder="Describe your issue or share your ideas" required></textarea> <br>
+<button class="btn" (click)="send(feedbackInput.value, usernameInput.value)">Send</button>
```
Comment (kirjs) [inline src/app/feedback-widget/feedback-widget.component.html]: You're not using the class name, either use or remove

**Comment by kirjs** [comment]
beutiful!

**Pull request merged**